### PR TITLE
Jira/flc 213 litecoin lib

### PR DESCRIPTION
**Pull request by artemkanaki**
Litecoin lib was migrated

### CODE REVIEW

**Review by grimelion — APPROVED** [review]

File: packages/fluence.lib.bitcoin/config/test.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
     "multivest": {
+        "blockchain": {
+            "bitcoin": {
+                "providers": {
+                    "native": {
+                        "host": "94.130.216.246",
```
Comment (andapplctr) [inline packages/fluence.lib.bitcoin/config/test.json:7]: remove IP, and provide local env var instead

File: packages/fluence.lib.litecoin/package.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@fluencesh/fluence.lib.litecoin",
+  "version": "2.0.15-alpha.0",
+  "private": false,
+  "description": "Bitcoin Blockchain Plugin",
```
Comment (andapplctr) [inline packages/fluence.lib.litecoin/package.json:5]: Litecoin BLockchain plugin

File: packages/fluence.lib.litecoin/sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Required metadata
+sonar.organization=applicature
```
Comment (andapplctr) [inline packages/fluence.lib.litecoin/sonar-project.properties:2]: is it correct org?

File: packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    public getSymbol() {
+        return 'LTC';
+    }
+
+    public getServiceId() {
+        return `litecoin.${ this.getNetworkId() }.blockchain.service`;
+    }
+
+    public isValidNetwork(network: string) {
+        return AvailableNetwork.LITECOIN === network || AvailableNetwork.TEST_NET === network;
```
Comment (andapplctr) [inline packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts:37]: Litecoin Testnet?

File: packages/fluence.lib.services/package.json [hunk 5]
```diff
@@ -29,9 +29,10 @@
     },
     "license": "Apache-2.0",
     "dependencies": {
-        "@applicature/synth.metrics": "^2.0.18",
-        "@applicature/synth.mongodb": "^2.0.18",
-        "@applicature/synth.plugin-manager": "^2.0.18",
+        "@applicature/synth.dynamodb": "^2.0.21",
```
Comment (andapplctr) [inline packages/fluence.lib.services/package.json:32]: Do we use dynamodb?

File: packages/fluence.lib.services/src/dao/dynamodb/webhook.action.dao.ts [hunk 6]
```diff
@@ -0,0 +1,140 @@
+import { DynamoDBDao } from '@applicature/synth.dynamodb';
+import { Hashtable } from '@applicature/synth.plugin-manager';
+import { DaoCollectionNames, DaoIds } from '../../constants';
+import { Scheme } from '../../types';
+import { WebhookActionDao } from '../webhook.action.dao';
+
+export class DynamodbWebhookActionDao extends DynamoDBDao<Scheme.WebhookActionItem> implements WebhookActionDao {
```
Comment (andapplctr) [inline packages/fluence.lib.services/src/dao/dynamodb/webhook.action.dao.ts:7]: Do we use DynamoDB?

File: packages/fluence.lib.services/src/models/index.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import { attribute, hashKey, table } from '@aws/dynamodb-data-mapper-annotations';
+
+@table('Dao')
+export class WebhookDaoActionItem {
```
Comment (andapplctr) [inline packages/fluence.lib.services/src/models/index.ts:4]: Do we use DynamoDB?

**Review by andapplctr — CHANGES_REQUESTED** [review]
Check the ocmments

**Pull request closed without merging**